source/MaterialXGraphEditor/Graph.h

```diff
@@ -32,13 +32,64 @@ struct Link
     }
 };
 
+class NodeDefParameters
```

Comment:
Block of possible creation parameters filled in the ui and passed to the method to perform creation and save.